frontend/packages/kubevirt-plugin/src/components/create-vm/dev-console/dev-console-create-vm-form.tsx

```diff
@@ -0,0 +1,218 @@
+import * as React from 'react';
+import Helmet from 'react-helmet';
+import { RouteComponentProps } from 'react-router';
+import {
+  ActionGroup,
+  Button,
+  EmptyState,
+  Title,
+  EmptyStateBody,
+  Stack,
+  StackItem,
+} from '@patternfly/react-core';
+import { ButtonBar, StatusBox, history, ExternalLink } from '@console/internal/components/utils';
+import { getUID } from '@console/shared';
+import { isTemplateSourceError } from '../../../statuses/template/types';
+import { getTemplateSourceStatus } from '../../../statuses/template/template-source-status';
+import { filterTemplates } from '../../vm-templates/utils';
+import {
+  getTemplateName,
+  getTemplateProvider,
+  isCommonTemplate,
+} from '../../../selectors/vm-template/basic';
+import { createVM } from '../../../k8s/requests/vm/create/simple-create';
+import { CreateVMForm } from '../forms/create-vm-form';
+import { getTemplateOSIcon } from '../../vm-templates/os-icons';
+import { formReducer, initFormState } from '../forms/create-vm-form-reducer';
+import { useStorageClassConfigMap } from '../../../hooks/storage-class-config-map';
+import { SUPPORT_URL } from '../../../constants/vm-templates';
+import { useVmTemplatesResources } from '../hooks/use-vm-templates-resources';
+
+export const DevConsoleCreateVmForm: React.FC<RouteComponentProps> = () => {
+  const urlParams = new URLSearchParams(window.location.search);
+  const templateParam = urlParams.get('common-template');
```

Comment:
i'd call this
```suggestion
  const templateParam = urlParams.get('template');
```

since it doesnt have to be only common-template